pkg/sloop/webserver/webfiles/sloop_ui.js

```diff
@@ -6,13 +6,13 @@
  */
 
 const palette = {
-        baseDark: ["#2E3440", "#3B4252", "#434C5E", "#4C566A"],
-        baseLight: ["#D8DEE9", "#E5E9F0", "#ECEFF4", "#F3F3FE"],
-        primary: ["#8FBCBB", "#88C0D0", "#81A1C1", "#5E81AC"],
-        highlight: ["#BF616A", "#D08770", "#EBCB8B", "#A3BE8C",
-            "#B48EAD","#ADA8B6","#da7650","#D496A7","#ABC4AB",
-            "#A4A6D2","#91BEF2","#97B1A6","#8A9B68"],
-        severity: ['#4BD855', '#E0E000', '#D84B4B'],
+    baseDark: ["#2E3440", "#3B4252", "#434C5E", "#4C566A"],
```

Comment:
intentional indentation changes?

Reply:
Yes, other places in the code mostly used single tab so I changed it.